specification/src/specification/capabilities.md

```diff
@@ -22,8 +22,10 @@ See [`CapabilitiesResponse`](../reference/types.md#capabilitiesresponse)
 
 | Name | Description |
 |------|-------------|
-| `versions` | A [semantic versioning](https://semver.org) range of API versions which the data connector 
+| `versions` | A [semantic versioning](https://semver.org) range of API versions which the data connector
 | `capabilities.explain` | Whether the data connector is capable of describing query plans |claims to implement |
+| `capabilities.mutation` | Whether the data connector is capable of executing mutations |claims to implement |
```

Comment:
I think we can delete this line now that it's required anyway.

Reply:
done.